tests/rgb_tests.cpp

```diff
@@ -22,6 +22,8 @@
 
 #include <gtest/gtest.h>
 
+#include <cmath>
```

Comment:
Making changes to code should be in a separate commit after this. This commit should only do what the commit message says.